components/compliance-service/dao/pgdb/filtering.go

```diff
@@ -5,11 +5,12 @@ import (
 	"strings"
 	"time"
 
-	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
-
 	"github.com/chef/automate/components/compliance-service/api/common"
 	"github.com/chef/automate/components/compliance-service/utils"
+	"github.com/chef/automate/lib/errorutils"
+	"github.com/chef/automate/lib/pgutils"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
```

Comment:
I think we are trying to separate the internal go packages and the external one, also ordering them by name cc @srenatus @msorens since they have been fixing the multiple times with PRs like this one https://github.com/chef/automate/pull/1159